### ISSUE TITLE
Scoring Guide

### ISSUE THREAD

**Issue by keyserj** [amelioro/ameliorate#227]
**Is your feature request related to a problem? Please describe.**
There are lots of ways to interpret the scores:
- what does a 1 mean? what does a 5 mean? what does a 9 mean? what does a - mean?
- what do these mean when scoring a node? what about when scoring an edge?
- does the meaning different based on the node type or edge type?

**Describe the solution you'd like**
We should have a Guide page that explains the meanings of these, that can be used as a reference.

**Describe alternatives you've considered**
Could also call it the "How To" page

**Additional context**
Maybe should rename the "how to" section on the home page to "demo", and link to the guide.

**Technical ideas**

**Comment by keyserj** [comment]
Could also maybe add something like

> Instead of thinking in terms of pros and cons, "private property is good because [pros] and bad because [cons]", think in terms of criteria: "private property embodies criterion 1 well, criterion 2 poorly". Criterion enable solutions to be discussed and compared based on tradeoffs, which is better suited when more than two solutions are involved (or _can_ be involved). Thinking this way seems generally more open and cooperative.

**Comment by keyserj** [comment]
Added [a simple scoring explanation](https://ameliorate.app/docs/getting-started/diagramming-basics#scoring-to-convey-opinion) in the getting-started docs, seems sufficient for now
